### ISSUE TITLE
3-0：構成変更: 3章の冒頭に「仕様の要素説明の読み方」を追加する

### ISSUE THREAD

**Issue by bakera** [bakera/html-book#262]
Slackより:

>そうですね〜、3章で書かれている「この要素のARIAロールが◯◯になる」「この要素のふるまいはaria-*に相当する」の話がわかるのがほしい感じはあります。1章ではWCAGとかWAI-ARIAの存在の紹介に留めて、3章のアタマで「3章の読み方」みたいな節が一個あって、要素ごとの説明のパターンを紹介しつつ、WCAGとかWAI-ARIAとHTMLの絡みをもう少し書く、みたいな感じもいいのかもと思いました。 

>3章の読み方というよりもHTML仕様の要素説明の読み方が必要だと思っていて、実は1-6に仕様の読み方の話が入る想定でいたのですが、該当箇所をあらためて見ると概観的な内容になっていますね。3章から読み始めることも可能なようにしたいので、3章の頭に入れるか、1-6に入れて3章から参照するか…

**Comment by bakera** [comment]
#375 のインタラクティブも説明可能か?

**Comment by bakera** [comment]
Slackの議論:

1-7に「周辺技術の仕様を参照する」というセクションがあって
- ①は仕様の名前です。
- ②は仕様のステータスと、発行された日付です。
みたいなのがありますが、そんな感じで要素説明の読み方を説明するのがいいのかなと思っていました。

https://momdo.github.io/html/dom.html#element-definitions
を（改めて？）説明する感じです？
ただまあ、半分くらいはChapter2で触れてはいるって話もありますわね
ああでもスクショを取って、と思えば…

**Comment by momdo** [comment]
#444 で対応済みのためcloseします
